src/views/Leads/Grid.js

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Masonry from '#components/Masonry';
+import LoadingAnimation from '#rscv/LoadingAnimation';
+import { isArrayEqual } from '#rsu/common';
+import LeadItem from './GridItem';
+import styles from './styles.scss';
+
+const propTypes = {
+    loading: PropTypes.bool,
+    onEndReached: PropTypes.func.isRequired,
+    leads: PropTypes.arrayOf(PropTypes.object),
+};
+
+const defaultProps = {
+    loading: false,
+    leads: [],
+};
+
+export default class LeadGrid extends React.Component {
+    static propTypes = propTypes;
+    static defaultProps = defaultProps;
+
+    constructor(props) {
+        super(props);
+        this.ref = React.createRef();
+    }
+
+    render() {
+        const { loading, onEndReached, leads, ...otherProps } = this.props;
+
+        const columnWidth = 280;
+        const columnGutter = 40;
+
+        const items = leads.map(lead => (
+            {
+                lead,
+                minHeight: 295,
+                width: columnWidth,
+                ...otherProps,
+            }
+        ));
+
+        const compareItems = (a, b) => {
```

Comment:
This should be a static function or moved outside the class.
Creating new function in render method causes un-necessary rerenders